engine/src/main/java/org/terasology/rendering/dag/nodes/ShadowMapNode.java

```diff
@@ -81,6 +80,8 @@ public void initialise() {
 
         addDesiredStateChange(new BindFBO(SCENE_SHADOW_MAP));
         addDesiredStateChange(new SetViewportSizeOf(SCENE_SHADOW_MAP));
+        addDesiredStateChange(new Clear(true, true, false));
+        addDesiredStateChange(new DisableCapability(GL_CULL_FACE));
```

Comment:
Reviewing this again. I'm getting confused between enabling culling and the command that sets _what_ should be culled. I'm thinking if a StateChange _enables_ or _disables_ something we are dealing with a boolean parameter and we can call that state change `EnableSomething` or `DisableSomething`, depending on what's the default. The constructor parameters than can be just true or false, i.e.:

`new EnableFaceCulling(true)`

On the other hand a state change that sets a more complex type of parameter, i.e. what to cull, can be named SetSomething, i.e.:

`new SetFacesToCull(GL_BACK)`

I hope this clarifies things a little.